### PR TITLE
Add Indonesian locale & README improvement

### PR DESCRIPTION
**Pull request by okkidwi**

### CODE REVIEW

**Comment by pionxzh** [comment]
It looks great. Thanks for bringing the README to another level ❤️

**Comment by pionxzh** [comment]
Let me know if you have finished all the work 🙏

**Comment by pionxzh** [comment]
And for the contribution guide, I would like only to keep the English version. Please see https://github.com/pionxzh/chatgpt-exporter/pull/149#issuecomment-1570719316

**Comment by okkidwi** [comment]
> And for the contribution guide, I would like only to keep the English version. Please see https://github.com/pionxzh/chatgpt-exporter/pull/149#issuecomment-1570719316

Okay, I'll fix it now

**Comment by okkidwi** [comment]
> Let me know if you have finished all the work 🙏

Ok, i'm done👌

File: packages/userscript/src/i18n.ts [hunk 1]
```diff
@@ -35,6 +36,12 @@ const ES = {
     resource: es,
 }
 
+const ID_ID = {
+    name: 'Indonesia',
+    code: 'id-ID',
+    resource: Id_ID,
```
Comment (pionxzh) [inline packages/userscript/src/i18n.ts:42]:
```suggestion
    resource: id_ID,
```

File: packages/userscript/src/i18n.ts [hunk 2]
```diff
@@ -80,6 +88,9 @@ const LanguageMapping: Record<string, string> = {
     'es-CO': ES.code,
     'es-MX': ES.code,
     'es-US': ES.code,
+
```
Comment (pionxzh) [inline packages/userscript/src/i18n.ts:91]:
```suggestion

```

**Comment by pionxzh** [comment]
I will update the cover image later. The current one is not aligned with the latest UI.

Btw, for the star history, why not https://star-history.com/#pionxzh/chatgpt-exporter&Date ?

And I think the author label is not important 😆 want to remove it

**Comment by okkidwi** [comment]
> 

Okay, i'll fix it again

File: packages/userscript/src/i18n.ts (hunk 2 above)
Comment (pionxzh) [reply packages/userscript/src/i18n.ts:91]: These are lint fix

File: README.md [hunk 3]
```diff
@@ -1,38 +1,68 @@
...
+[last-commit-url]: https://github.com/pionxzh/chatgpt-exporter/commits
+[GreasyFork-image]: https://img.shields.io/static/v1?label=%20&message=GreasyFork&style=flat-square&labelColor=7B0000&color=960000&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH3ggEBCQHM3fXsAAAAVdJREFUOMudkz2qwkAUhc/goBaGJBgUtBCZyj0ILkpwAW7Bws4yO3AHLiCtEFD8KVREkoiFxZzX5A2KGfN4F04zMN+ce+5c4LMUgDmANYBnrnV+plBSi+FwyHq9TgA2LQpvCiEiABwMBtzv95RSfoNEHy8DYBzHrNVqVEr9BWKcqNFoxF6vx3a7zc1mYyC73a4MogBg7vs+z+czO50OW60Wt9stK5UKp9Mpj8cjq9WqDTBHnjAdxzGQZrPJw+HA31oulzbAWgLoA0CWZVBKIY5jzGYzdLtdE9DlcrFNrY98zobqOA6TJKHW2jg4nU5sNBpFDp6mhVe5rsvVasUwDHm9Xqm15u12o+/7Hy0gD8KatOd5vN/v1FozTVN6nkchxFuI6hsAAIMg4OPxMJCXdtTbR7JJCMEgCJhlGUlyPB4XfumozInrupxMJpRSRtZlKoNYl+m/6/wDuWAjtPfsQuwAAAAASUVORK5CYII=
+[GreasyFork-url]: https://greasyfork.org/scripts/456055-chatgpt-exporter
+    
+English 🇺🇸 &nbsp;&nbsp;|&nbsp;&nbsp; [Indonesia 🇮🇩](./README_ID.md) &nbsp;&nbsp;|&nbsp;&nbsp; [한국어 🇰🇷](./README_KR.md) &nbsp;&nbsp;|&nbsp;&nbsp; [Türkçe 🇹🇷](./README_TR.md)
```
Comment (pionxzh) [inline README.md:21]: Let's keep it no country flag

Comment (okkidwi) [reply README.md]: > Let's keep it no country flag

I've deleted it, if anything needs to be fixed just let me know

File: packages/userscript/src/i18n.ts (hunk 2 above)
Comment (pionxzh) [reply packages/userscript/src/i18n.ts:91]: @okkidwi  Can you click on the "Commit suggestion" to fix the lint here?

File: README.md (hunk 3 above)
Comment (pionxzh) [reply README.md]: Thanks. The reason for this is that I want to decouple the relationship between language and country.

File: packages/userscript/src/i18n.ts (hunk 1 above)
Comment (pionxzh) [reply packages/userscript/src/i18n.ts:42]: @okkidwi

File: packages/userscript/src/i18n.ts [hunk 4]
```diff
@@ -79,7 +87,9 @@ const LanguageMapping: Record<string, string> = {
     'es-CL': ES.code,
     'es-CO': ES.code,
     'es-MX': ES.code,
-    'es-US': ES.code,
+    'es-US': ES.code,
```
Comment (pionxzh) [inline packages/userscript/src/i18n.ts:90]:
```suggestion
    'es-US': ES.code,
```

**Comment by pionxzh** [comment]
Thank you!

**Pull request merged**